nototools/generate_website_2_data.py

```diff
@@ -554,7 +554,7 @@ def get_css_generic_family(family):
 
 CSS_WEIGHT_MAPPING = {
     'Thin': 250,
```

Comment:
In that case, ```'ExtraLight': 200``` is odd. Does ExtraLight have OS/2.fsWeight set to 200? 

I thought we use '250' in OS/2.fsWeight [1] because Windows GDI does not work well with anything lower than 250.  That being the case, ExtraLight with OS/2.fsWeight of 200 wouldn't work, either. No? 

[1] I'm not a fan of using 250 for Thin to make Windows GDI happy because Android, Chrome OS, Mac and Linux work perfectly well with 100.